demos/rerank/README.md

```diff
@@ -16,142 +16,92 @@ It will create an image called `openvino/model_server:latest`.
-
-
-## Server configuration
-Prepare config.json:
-```bash
-cat config.json
-{
-    "model_config_list": [
-        {
-            "config": {
-                "name": "tokenizer",
-                "base_path": "/workspace/models/BAAI/bge-reranker-large-tokenizer"
-            }
-        },
-        {
-            "config": {
-                "name": "rerank_model",
-                "base_path": "/workspace/models/BAAI/bge-reranker-large-rerank"
-            }
-        }
-    ],
-    "mediapipe_config_list": [
-        {
-            "name": "rerank",
-            "graph_path": "/workspace/models/graph.pbtxt"
-        }
-    ]
-}
+tree models
+models
+├── BAAI
+│   └── bge-reranker-large
+│       ├── graph.pbtxt
+│       ├── rerank
+│       │   └── 1
+│       │       ├── model.bin
+│       │       └── model.xml
+│       ├── subconfig.json
+│       └── tokenizer
+│           └── 1
+│               ├── model.bin
+│               └── model.xml
+└── config.json
 
 ```
+> **Note** The actual models support version management and can be automatically swapped to newer version when new model is uploaded in newer version folder.
 
 
-## Start-up
+## Deployment 
+
 ```bash
-docker run -d --rm -p 8000:8000 -v $(pwd)/:/workspace:ro openvino/model_server:latest --port 9000 --rest_port 8000 --config_path /workspace/config.json
+docker run -d --rm -p 8000:8000 -v $(pwd)/models:/workspace:ro openvino/model_server:latest --port 9000 --rest_port 8000 --config_path /workspace/config.json
 ```
 
 ## Client code
 
 
-```python
-import requests
-
-# Define the API endpoint and your API key
-api_endpoint = "http://localhost:8000/v3/rerank"
-api_key = "YOUR_API_KEY"
-
-# Create the payload
-payload = {
-    "model": "rerank",
-    "query": "what is it?",
-    "documents": [
-        "one text",
-        "two text longer than one"
-    ]
-}
-
-# Set up the headers
-headers = {
-    "Authorization": f"Bearer {api_key}",
-    "Content-Type": "application/json"
+```bash
+curl http://localhost:8000/v3/rerank  -H "Content-Type: application/json" \
+-d '{ "model": "BAAI/bge-reranker-large", "query": "welcome", "documents":["good morning","farewell"]}' | jq .
+```
+```json
+{
+  "results": [
+    {
+      "index": 0,
+      "relevance_score": 0.3886180520057678
+    },
+    {
+      "index": 1,
+      "relevance_score": 0.0055549247190356255
+    }
+  ]
 }
+```
 
-# Make the POST request
-response = requests.post(api_endpoint, json=payload, headers=headers)
 
-# Handle the response
-if response.status_code == 200:
-    reranked_results = response.json()
-    print("Reranked Results:", reranked_results)
-else:
-    print("Error:", response.status_code, response.text)
+## Comparison with Hugging Faces
 
+```bash
+python compare_results.py --query "hello" --document "welcome" --document "farewell"
```

Comment:
To run it in automatic tests we need to reference this file as i.e. demos/rerano/compare_results.py